### PR TITLE
Added clue_ams_remote.py

### PR DESCRIPTION
**Pull request by dherrada**
Was experimenting with using buttons and capacitive touch as a remote using ams to add to pyloton. I thought the test code I wrote could be quite useful, so I'm PRing it. Didn't really want to add displayio to keep it simple and increase responsiveness, although adding that wouldn't take a whole lot of time to do.

### CODE REVIEW

File: examples/clue_ams_remote.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+"""
+This example solicits that apple devices that provide notifications connect to it, initiates
+pairing, then allows the user to use a CLUE board as a media remote through both the buttons
+and capacitive touch pads.
+"""
```
Comment (kattni) [inline examples/clue_ams_remote.py]: I would prefer if you could include a note in the doc string about the separate libraries needed to run this example. Whenever I include an example in a library-specific repo that requires an atypical library (e.g. adafruit_ble, and adafruit_ble_apple_media), I include a note in the docstring to alleviate any potential confusion.

Comment (dherrada) [reply examples/clue_ams_remote.py]: Will do

**Comment by dherrada** [comment]
@kattni I made a version of this on a branch using displayio. It looks pretty cool, but it is probably a bit more complex than we'd want for an example. [Link if you're interested](https://github.com/dherrada/Adafruit_CircuitPython_CLUE/blob/ams_with_displayio/examples/clue_ams_remote.py)

**Comment by kattni** [comment]
@dherrada Make a folder in `examples` called `advanced_examples`, rename the advanced example to something other than the same as this one, and feel free to include it that way. There are "duplicate" examples on the Circuit Playground library that are more advanced versions of similar things, so we can do that here as well. You can include it in this PR if you'd like. I'll hold off on approving until you decide.

**Comment by dherrada** [comment]
@kattni I made the requested changes

**Pull request merged**